documentation/staging/content/quickstart/create-domain.md

```diff
@@ -2,57 +2,50 @@
-    {{% /notice%}}
+1. Create the `sample-domain1` domain using a domain resource. The domain resource does not replace the traditional domain configuration files, but instead cooperates with those files to describe the Kubernetes artifacts of the corresponding domain.
 
-    First, copy the sample [create-domain-inputs.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml) file and update your copy with:  
-       * `domainUID`: `sample-domain1`
-       * `image`: Leave empty unless you need to tag the new image that the script builds to a different name.
-          For example if you are using a remote cluster that will need to pull the image from a container registry,
-          then you should set this value to the fully qualified image name.  Note that you will need to
-          push the image manually.
-       * `weblogicCredentialsSecretName`: `sample-domain1-weblogic-credentials`
-       * `namespace`: `sample-domain1-ns`
-       * `domainHomeImageBase`: `container-registry.oracle.com/middleware/weblogic:12.2.1.4`
+   - Use the following command to apply the sample domain resource:
 
-    For example, assuming you named your copy `my-inputs.yaml`:
+       ```shell
+       $ kubectl apply -f https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/quick-start-domain-resource.yaml
+       ```
 
-    ```shell
-    $ cd kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image
-    ```
-    ```shell
-    $ ./create-domain.sh -i my-inputs.yaml -o /<your output directory> -u <username> -p <password> -e
-    ```
-    {{% notice note %}}You need to provide the same WebLogic domain administrator user name and password in the `-u` and `-p` options
-    respectively, as you provided when creating the Kubernetes Secret in Step 2.
-    {{% /notice %}}
+    - **NOTE**: Instead of running the previous `kubectl apply` command, you can download the WLS Domain YAML file to `/tmp/quickstart/quick-start-domain-resource.yaml` or similar, using the following command, and then apply the file using `kubectl apply -f /tmp/quickstart/quick-start-domain-resource.yaml`:
 
-    For the detailed steps that the `create-domain.sh` script performs, see [Domain Home In Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}).
+      ```shell
+      $ curl -m 120 -fL https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/quick-start-domain-resource.yaml -o /tmp/quickstart/quick-start-domain-resource.yaml
+      ```
```

Comment:
Minor: Consider changing this slightly to more closely mirror the step in the previous bullet.

**NOTE**: Instead of running the previous `kubectl apply` command, you can download the WLS Domain YAML file to `/tmp/quickstart/quick-start-domain-resource.yaml` or similar, using the following command:

```
$ curl -m 120 -fL https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/quick-start-domain-resource.yaml -o /tmp/quickstart/quick-start-domain-resource.yaml
```
Then apply the file using:
```
kubectl apply -f /tmp/quickstart/quick-start-domain-resource.yaml
```